nexus/task_managers/tests.py

```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
```

Comment:
Flake8, not used import. You may also delete this file and add it later with the corresponding unittests.